org.mwc.debrief.lite/.project

```diff
@@ -19,5 +29,6 @@
 	<natures>
 		<nature>org.eclipse.jdt.core.javanature</nature>
 		<nature>org.eclipse.m2e.core.maven2Nature</nature>
```

Comment:
Hey, @ayeshare - I don't really know anything about natures. Can you explain what adding the PluginNature does?

Reply:
This converts a regular java project to be a plugin project.  Doing this is required to be able to run this from the top level pom file.  The pom file will build the project as if it is an eclipse plugin and during the compile phase, the pom file also makes a call to the debrief.lite's ant script